README.md

```diff
@@ -772,8 +776,128 @@ Note that the id that is passed to the `SerializationConfig` is same as the `fac
 
 ## 3. Custom Serialization
 
+Hazelcast lets you to plug a custom serializer to be used for serialization of objects.
+
+Let's say you have an object `Musician` and you would like to customize the serialization. The reason could be you are not happy with the default serialization.
+
+```javascript
+function Musician(name) {
+    this.name = name;
+}
```

Comment:
This object must have a `hzGetCustomId` method that will return the custom serialization id(10 in this example) AFAIK.